[2022-08-06 19:38:33,293] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 19:38:33,309] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 19:38:33,309] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:33,309] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:33,309] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:33,335] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-04 00:00:00+00:00
[2022-08-06 19:38:33,342] {standard_task_runner.py:52} INFO - Started process 2445 to run task
[2022-08-06 19:38:33,346] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-04T00:00:00+00:00', '--job-id', '749', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpbko8vyvl', '--error-file', '/tmp/tmpsqv8ikz4']
[2022-08-06 19:38:33,346] {standard_task_runner.py:80} INFO - Job 749: Subtask second_task
[2022-08-06 19:38:33,433] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:33,536] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:00:00+00:00
[2022-08-06 19:38:33,537] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:33,537] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:33,551] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:33,552] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:33,553] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:33,588] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220404T000000, start_date=20220806T193833, end_date=20220806T193833
[2022-08-06 19:38:33,637] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:33,677] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:25,400] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 21:52:25,417] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 21:52:25,417] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:25,417] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:25,417] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:25,437] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-04 00:00:00+00:00
[2022-08-06 21:52:25,444] {standard_task_runner.py:52} INFO - Started process 2638 to run task
[2022-08-06 21:52:25,447] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-04T00:00:00+00:00', '--job-id', '749', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpr8ro18yo', '--error-file', '/tmp/tmp6k0bh4hj']
[2022-08-06 21:52:25,448] {standard_task_runner.py:80} INFO - Job 749: Subtask second_task
[2022-08-06 21:52:25,520] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:25,611] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:00:00+00:00
[2022-08-06 21:52:25,612] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:25,612] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:25,626] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:25,627] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:25,628] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:25,661] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220404T000000, start_date=20220806T215225, end_date=20220806T215225
[2022-08-06 21:52:25,699] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:25,730] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:51,138] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 21:57:51,155] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 21:57:51,156] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:51,156] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:51,156] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:51,175] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-04 00:00:00+00:00
[2022-08-06 21:57:51,182] {standard_task_runner.py:52} INFO - Started process 2432 to run task
[2022-08-06 21:57:51,186] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-04T00:00:00+00:00', '--job-id', '750', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpb2799chz', '--error-file', '/tmp/tmpz0xx3ejz']
[2022-08-06 21:57:51,187] {standard_task_runner.py:80} INFO - Job 750: Subtask second_task
[2022-08-06 21:57:51,284] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:51,436] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:00:00+00:00
[2022-08-06 21:57:51,437] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:51,438] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:51,461] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:51,462] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:51,462] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:51,511] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220404T000000, start_date=20220806T215751, end_date=20220806T215751
[2022-08-06 21:57:51,564] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:51,644] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:51,621] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 22:12:51,637] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 22:12:51,637] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:51,637] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:51,637] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:51,659] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-04 00:00:00+00:00
[2022-08-06 22:12:51,669] {standard_task_runner.py:52} INFO - Started process 2447 to run task
[2022-08-06 22:12:51,672] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-04T00:00:00+00:00', '--job-id', '751', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpshzk8td2', '--error-file', '/tmp/tmp1gzj39i0']
[2022-08-06 22:12:51,673] {standard_task_runner.py:80} INFO - Job 751: Subtask second_task
[2022-08-06 22:12:51,758] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:51,854] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:00:00+00:00
[2022-08-06 22:12:51,855] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:51,855] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:51,868] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:51,870] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:51,870] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:51,961] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220404T000000, start_date=20220806T221251, end_date=20220806T221251
[2022-08-06 22:12:52,004] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:52,041] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:12,597] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 22:24:12,613] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [queued]>
[2022-08-06 22:24:12,613] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:12,613] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:12,613] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:12,635] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-04 00:00:00+00:00
[2022-08-06 22:24:12,643] {standard_task_runner.py:52} INFO - Started process 2506 to run task
[2022-08-06 22:24:12,648] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-04T00:00:00+00:00', '--job-id', '754', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpksoe41so', '--error-file', '/tmp/tmps_laqbk6']
[2022-08-06 22:24:12,649] {standard_task_runner.py:80} INFO - Job 754: Subtask second_task
[2022-08-06 22:24:12,734] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-04T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:12,832] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:00:00+00:00
[2022-08-06 22:24:12,833] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:12,834] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:12,847] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:12,849] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:12,849] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:12,890] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220404T000000, start_date=20220806T222412, end_date=20220806T222412
[2022-08-06 22:24:12,938] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:12,996] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
